lib/DDG/Spice/SublimePackages.pm

```diff
@@ -24,8 +23,16 @@ my $skip = join "|", share('skipwords.txt')->slurp(chomp => 1);
 
 handle remainder => sub {
     return unless $_; 
+    # OS search filters
+    s/linux/:linux/g; 
+    s/windows/:win/g;
+    s/mac\s?os\s?x|os\s?x/:osx/g;
```

Comment:
Should probably convert searches with `win` or `osx` to `:win` and `:osx`

Also add `\b\ boundary ?